morelights/init.lua

```diff
@@ -115,9 +123,31 @@ if morelights.game == "mineclone2" then
     a.stick = "mcl_core:stick"
 end
 
+if morelights.game == "hades_revisited" then
+    a.glass = "hades_core:glass"
+    a.glass_pane = "hades_xpanes:pane_flat"
```

Comment:
HR doesn't seem to have xpanes, should this be regular glass?

Reply:
hades_xpanes is part of [Mods pack for Hades Revisited](https://content.minetest.net/packages/SFENCE/hades_extensionmods/).

Reply:
Oh, I see. I could use the panes from your modpack if available, but I don't want to make `hades_xpanes` a hard dependency.